script.module.resolveurl/lib/resolveurl/plugins/streamsb.py

```diff
@@ -35,11 +35,11 @@ class StreamSBResolver(ResolveUrl):
                'javplaya.com', 'ssbstream.net', 'p1ayerjavseen.com', 'sbthe.com', 'vidmovie.xyz',
                'sbspeed.com', 'streamsss.net', 'sblanh.com', 'tvmshow.com', 'sbanh.com', 'streamovies.xyz',
                'embedtv.fun', 'sblongvu.com', 'arslanrocky.xyz', 'sbchill.com', 'sbrity.com', 'sbhight.com',
-               'sbbrisk.com', 'gomovizplay.com']
+               'sbbrisk.com', 'gomovizplay.com', 'sbface.com']
     pattern = r'(?://|\.)(' \
               r'(?:view|watch|embed(?:tv)?|tube|player|cloudemb|japopav|javplaya|p1ayerjavseen|gomovizplay|stream(?:ovies)?|' \
               r'vidmovie)?s{0,2}b?' \
-              r'(?:embed\d?|play\d?|video|fast|full|streams{0,3}|the|speed|l?anh|tvmshow|longvu|arslanrocky|chill|rity|hight|brisk)?\.' \
+              r'(?:embed\d?|play\d?|video|fast|full|streams{0,3}|the|speed|l?anh|tvmshow|longvu|arslanrocky|chill|rity|hight|brisk|sbface)?\.' \
```

Comment:
change `sbface` to `face` in the regex line, there is `sb` already there earlier so applies to all the other domains there such as `rity` `hight` etc